### ISSUE TITLE
What happend to Audit – Get Last Update?

### ISSUE THREAD

**Issue by visuary** [akaskela/WorkflowElements#6]
It's no longer present

**Comment by akaskela** [comment]
This was merged with Audit - Get Recent Updates.

**Comment by visuary** [comment]
So you can no longer get the most recent Update, you have to get all recent updates within a fixed time period?  That's not good.

**Comment by akaskela** [comment]
You're right, that kind of sucks. My intention was to add a new integer parameter for the max number of updates to retrieve, so you'd be able to replace Get Last Update with Get Recent Updates with a max of 1. Looks like I got distracted before implementing that. I'm reopening this and will include it with the next release.

**Comment by visuary** [comment]
Thank you for acknowledging and appreciating the importance of this feature and thank you for developing continuing to support this excellent toolset.

**Comment by akaskela** [comment]
I'm going to reopen this issue until the fix is in, so it doesn't fall off my radar again.

**Comment by akaskela** [comment]
Resolved in version 3.2.1.0
